client/components/ZoopDetails.tsx

```diff
@@ -43,13 +54,28 @@ const ZoopDetails = ({ zoop }: ZoopProps) => {
             </IconButton>
           </Stack>
         </Stack>
-        <Card>
-          <CardContent>
-            <Stack direction="row" justifyContent="center" alignItems="start">
-              <Typography variant="body1">{zoop.content}</Typography>
-            </Stack>
-          </CardContent>
-        </Card>
+
+          <Card>
+            <CardContent>
+              {currentUser && currentUser.id === zoop.authorId && (
+                <Stack direction="row" justifyContent="end">
+                  <ButtonGroup>
+                    <UpdateZoopButton 
+                      zoop={zoop}
+                    />
+                    </ ButtonGroup>
+                </Stack>
+              )}
+              <Link
+                to={`/zoops/${zoop.id}`}
```

Comment:
Also this would mean only the content is clickable, which is a worse click area. What was the purpose of this change?

Reply:
It had to do with allowing the update button to be clickable because when it was rendering in the Zoop (which seemed like the best design choice), and the whole Zoop was a link, one couldn't click the update button. If it's a problem I figure out how to make it work.

Reply:
`stopPropagation()` actually didn't work for the button (after a few seconds, the Link would be fired and it would navigate to the single zoop page). But `preventDefault()` worked. I did have to use `stopPropagation()` in a Stack component that wrapped the `UpdateZoopDialog` so that dialog would close if you click off of it (it was getting stuck when I used `stopPropagation()` on the Dialog `onClick`.